sample/src/main/java/co/nimblehq/sample/ui/login/LoginViewModelFactory.kt

```diff
@@ -14,11 +12,7 @@ class LoginViewModelFactory : ViewModelProvider.Factory {
     @Suppress("UNCHECKED_CAST")
     override fun <T : ViewModel> create(modelClass: Class<T>): T {
         if (modelClass.isAssignableFrom(LoginViewModel::class.java)) {
-            return LoginViewModel(
-                loginRepository = LoginRepository(
-                    dataSource = LoginDataSource()
```

Comment:
so do we still need to keep LoginRepository and LoginDataSource?

Reply:
@luongvo Same above